CodingTracker.AnaClos/CodingTracker.AnaClos/Validation.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System.Globalization;
+
+namespace AnaClos.CodingTracker;
+
+public class Validation
+{
+    public bool Validate2Dates(string startTime,string endTime)
+    {
+        bool ok;
+        var codingSession = new CodingSession { StartTime = startTime, EndTime = endTime };
+        codingSession.CalculateDuration();
+        if (codingSession.Duration.TotalSeconds > 0)
```

Comment:
Here since the expression returns a boolean you can assign the variable directly: 
![image](https://github.com/user-attachments/assets/25d28671-9a82-4d1d-8be4-26c662e5a540)

You can also return it directly:
![image](https://github.com/user-attachments/assets/c2add239-11a8-4126-a78f-4820aa1e39f3)

Reply:
Thanks for helping me improve my code.